maps/wyrm/wyrm_define.dm

```diff
@@ -81,7 +79,6 @@
 		"Aether Atmospherics",
 		"Zeng-Hu Pharmaceuticals",
 		"Hephaestus Industries",
-		"Commonwealth of Ahdomai"
```

Comment:
What's this one?

Reply:
Last I checked Tajara were pretty nuked.